taca/nanopore/ont_transfer.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import shutil
+
+from taca.nanopore.nanopore import Nanopore
```

Comment:
`from taca.nanopore.nanopore import Nanopore` 😅

Reply:
Naming is hard 😅